mathics_scanner/data/operators.yml

```diff
@@ -168,15 +177,15 @@ AngleBracket:
 Apply:
   actual-precedence: 820
   Precedence-Function: 620
-  precedence: 626
+  precedence: 620
```

Comment:
I guess the criterium now is what Precedece[] says, isn't it?

Reply:
I am probably using the terms in the way that Robert Jacobson had intended. 

 Right now, "precedence" is what mathics-core uses as precedence, whether it is Precendence[] or not. 

Ideally, this would be the same as `Precedence[]` and to the extent we can make it that we should. Only in those cases where we find we can't, I envision the `Precedence-Function` to give the `Precedence[]` value. And where they are the same there is no need to include this field in the table duplicating the "precedence" value. 

Down the line "actual-precedence" can be removed.

However, right now I want to get something minimally working.  Unfortunately, just this is already a big change.  But a really good one! We will be much closer to providing everything the way it appears in WMA with reduced hand-coded values strewn about in Python code.

As I type this I have finally gotten Mathics-core to pass the regular tests using data from this YAML file. The manifest stuff will require more work. 

Obviously this code has to get merged before mathics-core. So that will happen soon.